src/QtUtils.cpp

```diff
@@ -23,6 +25,13 @@ namespace QtUtils {
 		return QString();
 	}
 
+	std::filesystem::path qstring_to_path(const QString &input) {
+#ifdef _WIN32
```

Comment:
```suggestion
#ifdef Q_OS_WIN
```

For consistency with the rest of the codebase.